src/plugins/shared_ux/public/components/empty_state/analytics_no_data_page.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React from 'react';
+
+import { SharedUxServicesProvider } from '@kbn/shared-ux-services';
+
+import { getSharedUXServices } from '../../plugin';
+import { AnalyticsNoDataPageComponent } from './analytics_no_data_page.component';
+
+export interface AnalyticsNoDataPageProps {
+  onDataViewCreated: (dataView: unknown) => void;
+}
+
+export const AnalyticsNoDataPage = ({ onDataViewCreated }: AnalyticsNoDataPageProps) => {
+  const services = getSharedUXServices();
+  const { kibanaGuideDocLink } = services.docLinks;
+  return (
+    <SharedUxServicesProvider {...services}>
```

Comment:
Consuming plugins need to do this.  Let me know if you need help working around this limitation, I can help.

Reply:
This is what I was trying to avoid. If each plugin needs to provide their own service contract and this can't be "plug-n-play" component, then I agree that this component is not much needed, as all it does is provide a pre-initialized config.